.github/workflows/delete-stale-branches-preview.yml

```diff
@@ -0,0 +1,20 @@
+name: Delete stale visual snapshot branches Preview
+
+on:
+  # Allow workflow to be manually run from the GitHub UI
+  workflow_dispatch:
+jobs:
+  cleanup_visual_snapshot_branches:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete visual snapshot branches
+        uses: phpdocker-io/github-actions-delete-abandoned-branches@v2
+        with:
+          github_token: ${{ github.token }}
+          last_commit_age_days: 120
+          allowed_prefixes: visual-snapshot-update/pr-
+          ignore_branches: main,next,beta,15.x,13.x
```

Comment:
Is this needed? These branches don't match the `allowed_prefixes` anyway so this seems like something we don't need to maintain. If it's not needed, I think it should just be removed for simplicity.

Reply:
Okay, that's fine.